Multisig/Cross-layer/Configuration/ConfigurationKey.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  Configuration.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 27.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+/// Allows to fetch a variable from the main Info.plist or override it
+/// with a concrete value.
+///
+/// All of the standard data types present in the Plist format
+/// are supported as well, plus the URL type that is converted from String
+/// value.
+///
+/// The supported values conform to the `InfoPlistValueType` protocol.
+/// You can conform your own types to this protocol.
+///
+/// Example usage:
+///
+///     struct Configuration {
+///
+///         @ConfigurationKey("TERMS_URL")
+///         var termsURL: URL
+///
+///         @ConfigurationKey("RELAY_SERVICE_URL")
+///         var relayServiceURL: URL
+///
+///     }
+///
+/// IMPORTANT: test your configuration values, otherwise the app will
+/// crash if the value with the specified key is not found or if the value
+/// cannot be converted to the supported type.
+///
+@propertyWrapper
+struct ConfigurationKey<T: InfoPlistValueType> {
```

Comment:
Maybe it's better to define it in more general to read value from whatever plist file, I assume we might benefit from that later when we have firebase or tracking services